### PR TITLE
Adjust tutorials

### PR DESCRIPTION
**Pull request by eroell**
See https://github.com/theislab/ehrapy/issues/811 on ehrapy repo

### CODE REVIEW

File: mimic_2_fate.ipynb [hunk 1]
```diff
@@ -5,7 +5,7 @@
    "id": "70954479-ed2d-4bdb-90ca-5d607019e5b9",
```
Comment (eroell) [inline mimic_2_fate.ipynb:5]: <p>Should be double checked if the original description is still up to date regarding cluster numbers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (eroell) [reply mimic_2_fate.ipynb:5]: who should I ask here @Zethson ?

Comment (Zethson) [reply mimic_2_fate.ipynb:5]: Me? idk^^ the dataset is simple and we just look at it ourselves. Did the numbers look wrong, change or are fishy?

Comment (eroell) [reply mimic_2_fate.ipynb:5]: they look fishy to me, and I wonder if the leiden clusters changed from when this text was written

**Review by Zethson — APPROVED** [review]
I don't have a better idea for linking it. Submodule is too heavy because of all of the rest that we don't care about. Alternatively, we would link it.

File: mimic_2_fate.ipynb (hunk 1 above)
Comment (Zethson) [reply mimic_2_fate.ipynb:5]: Totally possible. But unfortunately, it'd be you (us) that has to look at them. No expert involved.

**Pull request merged**